lab_3_keywords_textrank/main.py

```diff
@@ -120,7 +130,11 @@ def _learn_indices(self, tokens: tuple[str, ...]) -> None:
             tokens : tuple[str, ...]
                 sequence of string tokens
         """
-        pass
+        self._word2id = {token: randint(1000, 9999) for token in tokens}
+        for index, token in enumerate(self._word2id):
+            w2i_values = list(self._word2id.values())
+            needed_value = w2i_values[index]
```

Comment:
why do not you use `self._word2id[token]` ?

Reply:
Done